app/viewmodels/checkAnswers/DateSubmittedSummary.scala

```diff
@@ -18,21 +18,27 @@ package viewmodels.checkAnswers
 
 import java.time.ZoneId
 import java.time.format.DateTimeFormatter
-import java.util.Locale
 
 import play.api.i18n.Messages
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 
+import com.ibm.icu.util.ULocale
```

Comment:
Is there an advantage to using `com.ibm.icu.util.ULocale`. Did a quick edit to still use `java.util.Locale` seemed to produce same result?